cpp/src/ast/operators.cpp

```diff
@@ -272,6 +272,68 @@ cudf::size_type ast_operator_arity(ast_operator op)
   return result;
 }
 
+std::string_view ast_operator_string(ast_operator op)
+{
+#define CUDF_AST_OP_STR(op) \
+  case ast_operator::op: return #op
+
+  switch (op) {
+    CUDF_AST_OP_STR(ADD);
```

Comment:
This is confusing to read since the `return` is part of the macro.
Perhaps we just get rid of the macro and write out each line?

Reply:
sure